### DISCUSSION TITLE
Coin types not matching up with slip44

### DISCUSSION THREAD

**Discussion by rdong8** [trustwallet/wallet-core#4169] [Q&A] [answered]
I notice that some of the derivation paths in `registry.json` have a coin type that doesn't match up with the slip44 coin type. Seems to be mainly Ethereum L2s using Ethereum's coin type instead of their own:

- opBNB uses 60 instead of 9006
- Base uses 60 instead of 8453
- Blast uses 60 instead of 342
- Scroll uses 60 instead of 534352

Why is this done?

**Comment by satoshiotomakan** [comment, marked as answer]
Hi @rdong8, thanks for asking. That's because WalletCore is mainly a network register for TrustWallet - a multi-chain wallet, and we target backward compatibility with other Ethereum wallets, for example, Metamask, so users can import their wallets to Trust, and get access to all L2 chains.
Note that MetaMask uses 60 slip44 coin type in the derivation path for all Ethereum chains
